### PR TITLE
.njk is interpreted as a nunjuck template

### PR DESCRIPTION
**Pull request by renevds**
.njk files are now interpreted as nunjuck files see #75

### CODE REVIEW

File: lib/utils.js [hunk 1]
```diff
@@ -3,6 +3,11 @@
 const path = require('path');
 const objectPath = require('object-path');
 
+//set of extensions that should be replaced
```
Comment (pheyvaer) [inline lib/utils.js:6]: Nit: space after `//`

File: test/parsers/html-parser.js [hunk 2]
```diff
@@ -4,14 +4,18 @@ const YAML = require('yaml');
...
 
 describe('HtmlParser', function () {
   {
     before(function () {
       const file = fs.readFileSync(path.resolve(__dirname, CONFIG_FILE), 'utf8');
       const yamlData = YAML.parse(file);
-
       this.output = parseHTML(yamlData.paths['/movies/{actor}']['get'].responses, '', '');
+
+      const njkFile = fs.readFileSync(path.resolve(__dirname, NJK_CONFIG_FILE), 'utf8');
```
Comment (pheyvaer) [inline test/parsers/html-parser.js:16]: Do other tests also need this?

Comment (renevds) [reply test/parsers/html-parser.js:16]: No, do I load this inside of the specific test?

Comment (pheyvaer) [reply test/parsers/html-parser.js:16]: That would be best then yes.

**Comment by pheyvaer** [comment]
LGTM. Can you make it up to date with the current state of `development`?

**Pull request merged**